lib/models/room.js

```diff
@@ -322,6 +322,21 @@ Room.prototype.getUsersReadUpTo = function(event) {
 };
 
 /**
+ * Get the ID of the event that a given user has read up to, or null if we
+ * have received no read receipts from them.
+ * @param {String} user ID to get read receipt event ID for
+ * @return {String} ID of the latest event that the given user has read, or null.
+ * an empty list.
+ */
+Room.prototype.getEventReadUpTo = function(userId) {
+    if (this._receipts === undefined || this._receipts['m.read'] === undefined) {
+        return null;
+    }
+
+    return this._receipts['m.read'][userId].eventId;
```

Comment:
`this._receipts['m.read'][userId]` may be `undefined` if they have never acknowledged a receipt.